### PR TITLE
Decision Reviews API - Add Show Endpoint For HigherLevelReviews

### PR DESCRIPTION
**Pull request by edmangimelli**
Add  _Show_ endpoint for Higher-Level Reviews. This can be used after creating a Higher-Level Review to check the status, see what was originally submitted in the _create payload_, and to retrieve any other information (created / updated time, for example).

Acceptance Criteria
- [x] working show endpoint
- [x] new tests
- [x] documentation

### CODE REVIEW

**Comment by edmangimelli** [comment]
> @edmangimelli I think should this endpoint reach out to CentralMail and get the status of the upload. else we are not showing accurate statuses

sounds good.
should that be a separate ticket, or should i roll that into this one?

File: modules/appeals_api/app/controllers/appeals_api/v1/decision_review/higher_level_reviews_controller.rb [hunk 1]
```diff
@@ -87,4 +92,26 @@ def model_errors_to_json_api
...
+      json: {
+        errors: [
+          { status: 404, detail: "HigherLevelReview with uuid #{@id.inspect} not found." }
+        ]
+      }
+    )
+  end
+
+  def render_higher_level_review
+    render json: AppealsApi::HigherLevelReviewSerializer.new(@higher_level_review).serialized_json
```
Comment (charleystran) [inline modules/appeals_api/app/controllers/appeals_api/v1/decision_review/higher_level_reviews_controller.rb:115]: in our other controller we follow the `render json: @higher_level_review, serializer: AppealsApi::HigherLevelReviewSerializer` style of getting the serialized response. More of a stylistic thing so I will leave that up to you

Comment (edmangimelli) [reply modules/appeals_api/app/controllers/appeals_api/v1/decision_review/higher_level_reviews_controller.rb:115]: i need that `.serialized_json` method when using fastjsonapi

Comment (edmangimelli) [reply modules/appeals_api/app/controllers/appeals_api/v1/decision_review/higher_level_reviews_controller.rb:115]: but maybe that would be automatically done for me?
i based mine off of the fast_jsonapi readme ¯\_(ツ)_/¯

**Pull request merged**